forge/forge/components/image_gen/test_image_gen.py

```diff
@@ -4,17 +4,18 @@
 from unittest.mock import patch
 
 import pytest
+from PIL import Image
+from pydantic import SecretStr
+
 from forge.components.image_gen import ImageGeneratorComponent
 from forge.components.image_gen.image_gen import ImageGeneratorConfiguration
 from forge.file_storage.base import FileStorage
 from forge.llm.providers.openai import OpenAICredentials
-from PIL import Image
-from pydantic import SecretStr
 
 
 @pytest.fixture
 def image_gen_component(storage: FileStorage):
-    cred = OpenAICredentials.from_env()
+    cred = OpenAICredentials(api_key=SecretStr("test"))
```

Comment:
This only works once the requests are cached in VCR. To get the requests in there, it will have to be run with an actual API key first.

Pls revert. :)

Reply:
`.from_env()` fails if there's no api key in the env but the test should still pass if there's vcr cassette, so I did:
```py
try:
    cred = OpenAICredentials.from_env()
except ValidationError:
    cred = OpenAICredentials(api_key=SecretStr("test"))
```
Is that ok or should I skip the test?

Reply:
That's a good solution, could you put that in a fixture if it's used more than once?